scipy/stats/_discrete_distns.py

```diff
@@ -1537,6 +1540,140 @@ def _rvs(self, a, size=None, random_state=None):
                         name='dlaplace', longname='A discrete Laplacian')
 
 
+class poisson_binom_gen(rv_discrete):
+    r"""A Poisson Binomial discrete random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    binom
+
+    Notes
+    -----
+    The probability mass function for `poisson_binom` is:
+
+    .. math::
+
+     f(k; p_1, p_2, ..., p_n) = \sum_{A \in F_k} \prod_{i \in A} p_i \prod_{j \in A^C} 1 - p_j
+
+    where :math:`k \in \{1, 2, \dots, n-1, n\}`, :math:`F_k` is the set of all
```

Comment:
I think in general here $k$ could be zero.
Also $k=0$ is an acceptable input to the pmf function.

Reply:
Yes, thanks.
```suggestion
    where :math:`k \in \{0, 1, \dots, n-1, n\}`, :math:`F_k` is the set of all
```